### DISCUSSION TITLE
Nuova Dashboard  Home Assistant- Energy

### DISCUSSION THREAD

**Discussion by Pask65** [indomus/forum#1303] [HUB: Home Assistant]
Recentemente home assistant ha integrato una dashboard per il calcolo dei consumi con grafici e widget dedicati.
Nella mia configurazione avevo già precedentemente installato lo Shelly EM via MQTT seguendo le indicazioni fornite dall'articolo https://indomus.it/guide/integrare-shelly-em-a-home-assistant-via-mqtt/ .
Ho provato a configurare la dashboard pensando di trovare disponibile il sensore Shelly EM ma la dropdownlist risulta vuota.
Avete qualche suggerimento ? Devo ripartire da zero disinstallando lo Shelly EM ?

Grazie

**Comment by Scialla** [comment]
Per essere visibile nel pannello energy il sensore deve avere `device_class: energy` e `state_class: total_increasing`, oppure deve essere creato con la piattaforma `utility_meter`.
Controlla e, se non li ha, aggiungili

**Reply by Scialla** [reply]
ah, controlla anche che l'entità sia nel recorder!

**Comment by Pask65** [comment]
Di seguito la mia configurazione che avevo e non ho cambiato della piattaforma utility_meter e recorder.
ma non vedo il sensore nella configurazione della dashboard Energy.
Dove sbaglio ?

utility_meter:
  home_weekly_power:
    source: sensor.power_2
    cycle: weekly
    tariffs:
      - F1
      - F2
  home_monthly_power:
    source: sensor.power_2
    cycle: monthly
    tariffs:
      - F1
      - F2
  home_hourly_energy:
    source: sensor.energy_2
    cycle: hourly
    tariffs:
      - F1
      - F2
  home_daily_energy:
    source: sensor.energy_2
    cycle: daily
    tariffs:
      - F1
      - F2
  home_weekly_energy:
    source: sensor.energy_2
    cycle: weekly
    tariffs:
      - F1
      - F2
  home_monthly_energy:
    source: sensor.energy_2
    cycle: monthly
    tariffs:
      - F1
      - F2

recorder:
  purge_keep_days: 7
  exclude:
    domains:
      - sensor
      - script    
      - automation
  include:
    domains:
      - light
      - switch
      - binary_sensor
    entities:
      - sensor.power_2_filtered
      - sensor.home_monthly_energy

**Reply by Scialla** [reply]
Cosa sbagli?
1 - a non usare il markdown, non si capisce una fava
2 - hai messo solo home_monthly_energy nel recorder, quindi solo quello potrai vedere nel pannello energia, gli altri sono esclusi

poi scusa se te lo chiedo ma che senso ha avere uno storico delle accensioni delle luci e degli switch e dei binary e non averlo dei sensori? Semmai il contrario..

Comunque uno dovresti vederlo, ma ci va un attimo se lo hai appena inserito nel recorder
